codegen/templates/interfaces/IYieldSource.sol.jinja

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
We should always use Apache-2.0. We can write interfaces by hand if we need to. It's important that we don't bring any copy-left code into the Hyperdrive codebase.

Reply:
roger, this was copied from our IER4626.sol, should we update there as well?